specifications/providers/openstack/flavors_and_images.md

```diff
@@ -126,6 +126,13 @@ As defined by this specification we expect:
 ```json
 {
   "properties": {
+    "unikorn:os:kernel": "linux",
+    "unikorn:os:familiy": "debian",
+    "unikorn:os:distro": "ubuntu",
+    "unikorn:os:variant": "server",
+    "unikorn:os:codename": "Noble Numbat",
+    "unikorn:os:version": "24.04",
+    "unikorn:packages": "slurmd=24.11,wireguard=1.0",
```

Comment:
One more point, perhaps we could consider the following?

```json
{
  "unikorn:package:kubernetes": "x.y.z",
  "unikorn:package:slurmd": "x.y.z"
}
```

Then we can unify everything.  Again it removes the need for parsing the actual value, which is just a version string.

Oh and, last but not least, the change log will need updating at the top of the file.  If we're getting rid of the `kubernetes_version` thing it'll be a "major" bump, so 2.0.0.